packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/tests/__snapshots__/gux-form-field-color.e2e.ts.snap

```diff
@@ -3,9 +3,11 @@
 exports[`gux-form-field-color #render help should render component as expected 1`] = `"<gux-form-field-color hydrated=""> <input slot="input" type="color" name="e-1" value="#cc3ebe" id="gux-form-field-input-i" aria-describedby="gux-form-field-help-i"> <label slot="label" for="gux-form-field-input-i">Default</label> <span slot="help" id="gux-form-field-help-i">This is a help message</span> </gux-form-field-color>"`;
 
 exports[`gux-form-field-color #render help should render component as expected 2`] = `
-<div class="gux-beside gux-form-field-container">
-  <div class="gux-beside gux-form-field-label">
+<div class="gux-above gux-form-field-container">
+  <div class="gux-above gux-form-field-label">
```

Comment:
I'm not 100% sure why this has happened in a few places. Looking into it.

Reply:
Fixed this, was an issue with the way that the spacing was applied in gux-form-field-label scss file